home.html

```diff
@@ -0,0 +1,358 @@
+                <p>Industry mentors will be standing by at the event to help individual teams brainstorm, debug, and
+                    polish their projects on an as-needed basis. Many of the mentor volunteers are UW Bothell alumni,
+                    and this year’s cohort draws employees from the following companies:
+                </p>
+                <div class="flex-columns">
+                    <div><ul><li>Microsoft</li><li>Google</li></ul></div>
+                    <div><ul><li>Amazon</li><li>Expedia</li></ul></div>
+                   <div><ul><li>T-Mobile</li><li>And more!</li></ul></div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <!--   Card 3 Content Slide Navigation -->
+    <div id="myNav3" class="overlay3">
+        <div class="flex-container">
+            <a href="javascript:void(0)" class="closebtn" onclick="closeNav3()">&times;</a>
+            <div class="overlay3-content">
+                <div class="image_card3">
+                    <img class="responsive" src="images/parts1.jpg" alt="Circuit Board Image">
+                </div>
+            </div>
+            <div><h1>Event Schedule</h1>
+                <table class="schedule">
+                    <tr><th>9:00</th><th>Check-In BeginsCheck-In Begins</th></tr>
+                    <tr><th>9:15</th><th>Free Breakfast </th></tr>
+                    <tr><th>10:00</th><th>Opening Ceremony</th></tr>
+                    <tr><th>10:30</th><th>Team Formation Mixer, hosted by ACM-W</th></tr>
+                    <tr><th>10:30</th><th>Hacking Begins</th></tr>
+                    <tr><th>11:15</th><th>Introduction To Cloud Services</th></tr>
+                    <tr><th>12:30</th><th>Free Lunch</th></tr>
+                    <tr><th>1:15</th><th>Lightning Talks: Tales from the Cloud</th></tr>
+                    <tr><th>2:45</th><th>Workshop TBA</th></tr>
+                    <tr><th>6:00</th><th>Free Dinner</th></tr>
+                    <tr><th>8:00</th><th>Submissions Due on DevPost</th></tr>
+                    <tr><th>8:15</th><th>Ice Cream Sundae Bar</th></tr>
+                    <tr><th>8:45</th><th> Demo Presentations</th></tr>
+                    <tr><th>9:45</th><th>Awards Ceremony</th></tr>
+
+                </table>
+            </div>
+        </div>
+        <div class="flex-container">
+            <div><h3>Content 2</h3> <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.</p></div>
```

Comment:
Changes to the master branch are instantly deployed to the static gh pages branch. We should not deploy the site when it still has lorem ipsum.